### PR TITLE
fix(global): :bug: fix CoreBackDrop validProps

### PR DESCRIPTION
**Pull request by PritamIT2023**
fix CoreBackDrop validProps

Ref: #333

## Description

Clearly describe the changes you've made in this pull request. Explain the purpose and reasoning behind the changes. Reference any relevant issues or discussions using keywords like "Fixes #<issue_number>" or "Resolves #<issue_number>".

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: package/components/feedback/CoreBackdrop.js [hunk 1]
```diff
@@ -107,12 +107,7 @@ CoreBackdrop.validProps = [
   {
     description: "The duration for the transition, in milliseconds. You may specify a single timeout for all transitions, or individually with an object.",
     name       : "transitionDuration",
-    types      : [
-      {
-        default: "Fade",
-        type   : "elementType",
-      }
-    ]
+    types      : [{ type: "number" }, { type: "object", validValues: "{ appear?: number, enter?: number, exit?: number }" }]
```
Comment (techoneel) [inline package/components/feedback/CoreBackdrop.js]: Valid values e type jay na

**Review by techoneel — CHANGES_REQUESTED** [review]
Require changes

File: package/components/feedback/CoreBackdrop.js (hunk 1 above)
Comment (PritamIT2023) [reply package/components/feedback/CoreBackdrop.js]: fixed

**Review by techoneel — APPROVED** [review]
good to go

**Pull request merged**